File: intermediate/challenge-1/app/models/bus.py
from sqlalchemy import Column, Integer, String
from db.base_class import Base


class Bus(Base):
    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
    capacity = Column(String, index=True)
    model = Column(String, index=True)
    make = Column(String, index=True)

File: intermediate/challenge-1/microservices/bus/src/main/java/com/beyond/microservice/bus/entity/Bus.java
package com.beyond.microservice.bus.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToOne;

import com.beyond.microservice.bus.driver.Driver;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Entity(name ="BUSTABLE")
@Getter
@Setter
@Builder
public class Bus {
    @Id
    @Column(name = "id", nullable = false)
    private Long id;
    private Integer capacity;
    private String model;
    private String maker;
    @OneToOne
    @JoinColumn(name = "driver_id")
    private Driver driver;
}
